api/routes/v1/sic_lookup.py

```diff
@@ -0,0 +1,68 @@
+"""Module that provides the SIC lookup endpoint for the Survey Assist API.
+
+This module contains the SIC lookup endpoint for the Survey Assist API.
+It defines the endpoint for looking up SIC codes based on descriptions.
+"""
+
+import sys
+
+from fastapi import APIRouter, HTTPException
+
+from api.services.sic_lookup_client import SICLookupClient
+
+router = APIRouter(tags=["SIC Lookup"])
+
+
+def get_lookup_client() -> SICLookupClient:
+    """Get a SIC lookup client instance.
+
+    Returns:
+        SICLookupClient: A SIC lookup client instance.
+    """
+    # Use test data during tests, knowledge base in production
+    if "pytest" in sys.modules:
```

Comment:
Hi - weirdly I recall making a comment about removing this and using the the test client logic you had added to conftest.py but maybe I imagined it because I can't find that comment anywhere?!

You did have a definition for test client:
def get_test_lookup_client() -> SICLookupClient:
        return SICLookupClient(data_path=str(sic_data_path))

And I was suggesting we try and use that pattern rather than this workaround.  Is that possible? If it is difficult getting going then we can live with this change.